### DISCUSSION TITLE
OM Android version

### DISCUSSION THREAD

**Discussion by pwb999** [organicmaps/organicmaps#2537] [Ideas]
Within the Bookmarks there is an option to Create New List. I would like to use this to organise my 100 or so imported KMLs files.

I would like one folder say "Map Data"; another for "Routes"; and another for "Aide-Memoirs" etc.
Then I would like to put my files into each of these Folders to organise them better.

For example:-
Map Data would have fences/walls/gates etc
Routes would have route1, route2, route3 etc
Aide-Memors would have FirstAid, Emergency Contacts, Kit list, Sun tables, Moon tables, Navigation Techniques etc

Is such functionality available?

Thanks

**Comment by biodranik** [comment]
It is possible in theory. But it's hard to make it easy for other users (maybe some "expert mode"?). Honestly, we have so many other issues asked by many users, and so little time to work on it, so it would be hard to prioritize this one.

**Reply by pwb999** [reply]
No problems. Thanks for your interest. I enjoy showing others just how useful OM is for hiking. Is there any possibiltiy that CRAGS may be incorporated into the Terrain layer. At the moment it is not safe to only use OM for hiking as many paths go through very serious and dangerous crags. For instance Thunderforest's Landscape has CRAGS information and that makes that map safer to use in the real world. If OM is trying to be useful to hikers and bikers then such features are really important.
<img width="801" alt="Screenshot 2022-05-09 at 18 47 26" src="https://user-images.githubusercontent.com/58628197/167604656-0e5b6623-2843-4c95-aa9c-d9d7a3574312.png">

Compared to OM it could be that turning on the TERRAIN Layer might also show this very important CRAG information.
<img width="777" alt="Screenshot 2022-05-10 at 11 08 25" src="https://user-images.githubusercontent.com/58628197/167605225-0f971bce-53a6-4a27-8a45-9ae6f4460b52.png">

Many thanks

**Reply by biodranik** [reply]
That's a good idea, can you please create a detailed issue about it, with screenshots and some links to OSM features? We probably need to update our renderer to draw it properly, but that's useful data, of course, if it's already in OSM.

**Comment by pwb999** [comment]
I have just extracted the natural=cliff tag IN Cumbria using overpass turbo and then opened it in the OM Macbook version. It would be useful at least to have this tag incorporated into the Terrain layer and styled in the same fashion as the above example. Thats for your interest.

<img width="769" alt="Screenshot 2022-05-10 at 11 25 31" src="https://user-images.githubusercontent.com/58628197/167608577-007fcf48-969f-4aad-b7e2-ba5971f4f1d8.png">
